client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -272,7 +272,7 @@ const PersonalPurchase = ( {
 			) : (
 				<div className={ `${ COMPONENT_CLASS_NAME }__actions` }>
 					<ButtonComponent
-						className="stats-purchase-page__full-width"
```

Comment:
I understand according to BEM rules, modifiers are supposed to look like `is-full-width`, but with the context of Odyssey, it would/have cause issues. CSS naming space is stuffed in Odyssey, unlike Calypso where it's just a standalone app. I would still lean to the original naming on this one if it makes sense. 🙂

Reply:
Generic CSS classes also tend to leak when navigating between Calypso pages. We have had several issues like this over the past few months.

Reply:
Okay, that makes sense. So for cases like this, we'd prefer to use a longer, more specific class name.